packages/dm-core-plugins/blueprints/list/ListPluginConfig.json

```diff
@@ -52,6 +52,13 @@
       "dimensions": "*",
       "default": [],
       "optional": false
+    },
+    {
+      "name": "readOnly",
```

Comment:
This is not needed. Whether or not list is readonly should be determined by the config.functionality. It isn't currently, because list is temporarily deprecated and no one works on it, but that is the idea of it at least.

Perhaps you could use the table plugin instead?